### PR TITLE
Move karaf distribution to brooklyn-dist.

### PR DESCRIPTION
**Pull request by geomacy**
This PR is the first of three that together move the karaf distribution to brooklyn-dist and separate out the software from brooklyn-library into separate catalog bom and feature files.

### CODE REVIEW

File: karaf/features/src/main/feature/feature.xml [hunk 1]
```diff
@@ -276,60 +276,6 @@
         <feature>jclouds-api-filesystem</feature>
 
         <bundle start-level="85">mvn:org.apache.brooklyn/brooklyn-locations-jclouds/${project.version}</bundle>
-        <feature>brooklyn-software-winrm</feature>
```
Comment (neykov) [inline karaf/features/src/main/feature/feature.xml]: `winrm` is part of core. Don't see where do you include the feature back (in -library and -dist PRs)?

**Comment by neykov** [comment]
LGTM (apart from the missing winrm dependency)

File: karaf/features/src/main/feature/feature.xml (hunk 1 above)
Comment (geomacy) [inline karaf/features/src/main/feature/feature.xml]: I had meant to move it to brooklyn-library and then realised it didn't belong there, must have missed putting it back here.  Will fix.

**Comment by geomacy** [comment]
Review comments applied, squashed and pushed.

**Comment by duncangrant** [comment]
I've tested this - all seems to work

**Pull request merged**